### PR TITLE
:recycle: Accepts new path parameters and updates tests

### PR DESCRIPTION
**Pull request by mitchdawson1982**

### CODE REVIEW

File: containers/daap-presigned-url/CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.2.4] 2023-10-03
+
+### Changed
+
+- removed datebase and table querystring parameters
```
Comment (murdo-moj) [inline containers/daap-presigned-url/CHANGELOG.md]:
```suggestion
- removed database and table querystring parameters
```

File: containers/daap-presigned-url/CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- created date-product-name & table-name variables from event body pathParameters
+- created body variable from event.body json string
+- replaced refferences to datebase with data-product-name
```
Comment (murdo-moj) [inline containers/daap-presigned-url/CHANGELOG.md]:
```suggestion
- replaced references to database with data-product-name
```

File: containers/daap-presigned-url/CHANGELOG.md [hunk 3]
```diff
@@ -9,6 +9,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.2.4] 2023-10-03
+
+### Changed
+
+- removed datebase and table querystring parameters
+- created date-product-name & table-name variables from event body pathParameters
```
Comment (murdo-moj) [inline containers/daap-presigned-url/CHANGELOG.md]:
```suggestion
- created data-product-name & table-name variables from event body pathParameters
```

File: containers/daap-presigned-url/CHANGELOG.md [hunk 4]
```diff
@@ -9,6 +9,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- created body variable from event.body json string
+- replaced refferences to datebase with data-product-name
+- replaced refferences to table with table-name
```
Comment (murdo-moj) [inline containers/daap-presigned-url/CHANGELOG.md]:
```suggestion
- replaced references to table with table-name
```

File: containers/daap-presigned-url/tests/unit/mock_test.py [hunk 5]
```diff
@@ -58,11 +64,15 @@ def test_dataproduct_does_not_exist(s3_client, fake_context, region_name, monkey
...
-        "queryStringParameters": {
-            "database": database,
-            "table": table,
-            "contentMD5": "3f92d72f7e805b66db1ea0955e113198",
-        }
+        "pathParameters": {
+            "data-product-name": database,
+            "table-name": table,
+        },
+        "body": json.dumps(
```
Comment (MatMoore) [inline containers/daap-presigned-url/tests/unit/mock_test.py]: Looking at the tests it seems like the API gateway must be passing the deserialized version of the body, and you can drop the `json.dumps` here.

Other than that all looks good to me 👍🏻

Comment (mitchdawson1982) [reply containers/daap-presigned-url/tests/unit/mock_test.py]: Thanks Mat. Yes I just ran into this in the tests and removed it. Many Thanks

**Review by murdo-moj — APPROVED** [review]

**Comment by tom-webber** [comment]
part of #1664

**Pull request merged**